macaroonbakery/codec.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+import base64
+import json
+import six
```

Comment:
six and nacl are not part of the stdlib.
So they should be placed in the second import group.